src/integrationTest/cloudformation/templateRegistryManager.test.ts

```diff
@@ -0,0 +1,136 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as path from 'path'
+import * as vscode from 'vscode'
+
+import { CloudFormationTemplateRegistry } from '../../shared/cloudformation/templateRegistry'
+import { CloudFormationTemplateRegistryManager } from '../../shared/cloudformation/templateRegistryManager'
+import { rmrf } from '../../shared/filesystem'
+import { getLogger, setLogger } from '../../shared/logger/logger'
+import { makeSampleSamTemplateYaml, strToYamlFile } from '../../test/shared/cloudformation/cloudformationTestUtils'
+import { TestLogger } from '../../test/testLogger'
+import { getTestWorkspaceFolder } from '../integrationTestsUtilities'
+
+/**
+ * Note: these tests are pretty shallow right now. They do not test the following:
+ * * Adding/removing workspace folders
+ */
+describe.only('CloudFormation Template Registry Manager', async () => {
```

Comment:
Remove .only

(@justinmk3 do you think we could somehow lint check for forgotten '.only' calls, and only check in CI but not locally? I forget these sometimes too)